ink/goddess/reward_joi

```diff
@@ -126,6 +127,25 @@ Would you like to wear a gag for me, {player_name}?
 
 + [distracted] -> Goddess_PayAttention -> mouth
 
+= lube
```

Comment:
Oooo! Very inspired. I like it!

Don't forget the [timeout] and [distracted] redirects need to go back to 'lube'